source/reference/minio-server/settings/core.rst

```diff
@@ -43,9 +43,9 @@ For example, to set up ftp access, you could set the variable to something like
 .. code-block:: shell
    :class: copyable
 
-   export MINIO_OPTS=' "--console-address="9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '
+   export MINIO_OPTS=' --console-address=":9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '
 
-On Unix-like systems, you can save a file with the environment variable to ``/etc/default/minio`` instead of setting the variable manually.
+On Unix-like systems using the recommended MinIO ``systemd`` service, you can save a file with the environment variable to ``/etc/default/minio`` instead of setting the variable manually. Note that the MinIO server does not read ``MINIO_OPTS`` directly, rather it is used to pass options via variable expansion when starting the MinIO server, e.g. ``minio server $MINIO_OPTS ...``.
```

Comment:
Mostly just formatting for consistency with the docs.
We also avoid latinisms like `via` and `e.g.`.

```suggestion
On Unix-like systems using the recommended MinIO ``systemd`` service, you can save a file with the environment variable to ``/etc/default/minio`` instead of setting the variable manually. 

.. note::

   The MinIO server does not read ``MINIO_OPTS`` directly.
   Instead, use the environment variable to pass options with variable expansion when starting the MinIO server.
   
   For example:
   
   .. code-block::
      :class: copyable

      minio server $MINIO_OPTS ...
```